operational/dbaas/aws/rds_backup/aws_rds_backup.pt

```diff
@@ -28,13 +42,21 @@ parameter "param_backup_window" do
   label "RDS Backup Window (HH:MM-HH:MM)"
 end
 
-auth "auth_aws", type: "aws" do
-  version 4
-  service 'rds'
-  access_key cred('AWS_ACCESS_KEY_ID')
-  secret_key cred('AWS_SECRET_ACCESS_KEY')
+###############################################################################
+# Authentication
+###############################################################################
+
+#authenticate with AWS
+credentials "auth_aws" do
+  schemes "aws"
```

Comment:
We need to also use the aws_sts scheme.  the schemes should be "aws","aws_sts"

Reply:
Updated the schemes part with "aws","aws_sts"